test/application/checks.test.js

```diff
@@ -0,0 +1,36 @@
+import 'babel-polyfill';
+import * as helpers from '../../src/checks';
+
+// TODO write all tests
```

Comment:
damn, that's a lot of work

Reply:
Oh yeah. I'm already excited about this.